resources/openshift-haproxy-router/dashboards.yaml

```diff
@@ -29,10 +29,10 @@ configurations:
     * Frontend Connections
     * Frontend Bytes Out
     * Frontend HTTP Requests
-  file: include/HAProxy_OC_Ingress_Overview-sysdig-dashboard.json
+  file: include/HAProxy_OC_Ingress_Overview-sysdig-dashboard-v4.7.json
 - name: 'HAProxy OC Service Golden Signals'
   kind: Sysdig
-  image: 'openshift-haproxy-router/images/HAProxy_OC_Service_Golden_Signals-sysdig-dashboard.png'
```

Comment:
Should we also delete the image from the 'images' folder?

Reply:
Yes, deleted